Gems/EMotionFX/Code/EMotionFX/Source/AnimGraph.cpp

```diff
@@ -33,6 +33,146 @@ namespace EMotionFX
 {
     AZ_CLASS_ALLOCATOR_IMPL(AnimGraph, AnimGraphAllocator)
 
+
+    namespace internal
+    {
+        static constexpr AZStd::array GraphNodeColors = { AZ::u32{ 0xFF000080 },
```

Comment:
I'm assuming that this array of colors was preselected and predefined elsewhere instead of actually being randomly generated generated?

Reply:
Yeah I had the same question at first, then I realized it was copied from the `MCore::s_colorTable` which is why I suggested it be removed as well so it's more clear where that came from.